src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -354,7 +353,11 @@ public Task InitializeAsync(DatabaseRecord record)
                 _initialized = true;
             }
 
-            return Task.Factory.StartNew(HandleDatabaseRecordChange, TaskCreationOptions.LongRunning);
+            return Task.Factory.StartNew(() =>
+            {
+                OpenIndexes(record);
```

Comment:
Do we really need this, won't just `HandleDatabaseRecordChange` do the same thing here?